Tests/UnitTests/FileShareService.DesktopClient.CoreTests/AuthProviderTests.cs

```diff
@@ -14,14 +15,15 @@ internal class AuthProviderTests
         private IEnvironmentsManager fakeEnvironmentsManager = null!;
         private INavigation fakeNavigation = null!;
         private IJwtTokenParser fakeJwtTokenParser = null!;
+        private readonly ILogger<AuthProvider> logger;
 
         [SetUp]
         public void Setup()
         {
             fakeEnvironmentsManager = A.Fake<IEnvironmentsManager>();
             fakeNavigation = A.Fake<INavigation>();
             fakeJwtTokenParser = A.Fake<IJwtTokenParser>();
-            authProvider = new AuthProviderForTesting(fakeEnvironmentsManager, fakeNavigation, fakeJwtTokenParser);
+            authProvider = new AuthProviderForTesting(fakeEnvironmentsManager, fakeNavigation, fakeJwtTokenParser,logger);
```

Comment:
Please set `logger` to `A.Fake<ILogger<AuthProvider>>()` to avoid null ref errors in the future

Reply:
Resolved